docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,474 @@
+{
```

Comment:
<blockquote><strong>If two or more discords in same dataset are, what can we do?</strong></blockquote><p> </p><p>I think this sentence is ungrammatical. The verb of the <code>if clause</code> is "are". Hence, you may revise it as follows: "If two or more discords <strong>are</strong> in a dataset, what can we do?"</p><p> </p><p>But, still, this feels weird to me. we may re-write it as:</p><p>"If a dataset has more than one discord, what can we do?"</p><p> </p><p><strong>When you ask the question "what can we do?", it is still <em>not</em> clear. </strong>Because I do not know what the goal is at this point. For instance: "If weather is not good, what can we do?" is not clear because it does not show the main goal and its challenge here. It can be anything... for example (1) Do we have a flight and because of weather our flights got cancelled? In this case "what can we do?" means "how can we adjust our trip?" (2) Do we have a plan to play soccer outside? Then "what can we do?" can mean what other games we can play in this weather (maybe indoor games). (3) etc.</p><p> </p><p>As you can see, a story should be clear. So, I think you should provide further explanation here.</p><p> </p><p> </p><blockquote>Let's demonstrate it in ECG data with anormaly behavior.</blockquote><p> </p><p>can you help me understand what <code>it</code> is referring to here? is that challenge?</p><p> </p><p>Also: I think <code>anomaly</code> is not adjective.  It is a noun. So, please remove the term behavior.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>